sandbox/grommet-app/src/theme.jsx

```diff
@@ -380,6 +380,32 @@ const buildTheme = tokens => {
     },
   };
 
+  const containerTokens = 'container' in large.hpe.size;
```

Comment:
This is for backwards compatibility in case the grommet-app is using an earlier version of tokens? Am I understanding correctly?

Reply:
Yes exactly.